contrib/platform/src/com/sun/jna/platform/win32/COM/Wbemcli.java

```diff
@@ -217,11 +217,11 @@ public HRESULT ConnectServer(BSTR strNetworkResource, BSTR strUser, BSTR strPass
 
         public IWbemServices ConnectServer(String strNetworkResource, String strUser, String strPassword,
                 String strLocale, int lSecurityFlags, String strAuthority, IWbemContext pCtx) {
-            BSTR strNetworkResourceBSTR = OleAuto.INSTANCE.SysAllocString(strNetworkResource);
-            BSTR strUserBSTR = OleAuto.INSTANCE.SysAllocString(strUser);
-            BSTR strPasswordBSTR = OleAuto.INSTANCE.SysAllocString(strPassword);
-            BSTR strLocaleBSTR = OleAuto.INSTANCE.SysAllocString(strLocale);
-            BSTR strAuthorityBSTR = OleAuto.INSTANCE.SysAllocString(strAuthority);
+            BSTR strNetworkResourceBSTR = OleAuto.INSTANCE.SysAllocString(strNetworkResource == null ? null : new WString(strNetworkResource));
```

Comment:
Not needed, `String` is handled properly.

Reply:
According to MSDN, OLECHAR is an alias for char, thus needs to be in ANSI format even on UNICODE API?